dist/index.js

```diff
@@ -1686,7 +1686,7 @@ function run() {
         try {
             let dsn = core.getInput("dsn", { required: true });
             yield core.exportVariable("SCOPE_DSN", dsn);
-            let pathVar = core.getInput("path", { required: true });
+            let pathVar = core.getInput("path", { required: true }).replace(/\n/g, " ");
```

Comment:
I'd use `/[\r\n]/gm` just to be safe. This is all combinations of `\n`, `\r` in multiple lines.

Also minor, if you don't intend to modify `pathVar` in this case, you'd prefer to use `const` instead of `let`

Reply:
Thanks mate